src/Compilation/Config.zig

```diff
@@ -48,6 +48,7 @@ use_lib_llvm: bool,
 use_lld: bool,
 c_frontend: CFrontend,
 lto: bool,
+lto_mode: LTOMode,
```

Comment:
Why don't we just merge these?

```suggestion
lto: LtoMode,
```